packages/patternfly-3/patternfly-react/src/components/TypeAheadSelect/TypeAheadSelect.stories.js

```diff
@@ -82,6 +94,7 @@ TypeAheadSelectStories.addWithInfo('Multiple Selection', () => {
       <br />
       <h2>Multiple Selection</h2>
       <TypeAheadSelect
+        id="some-typeahead"
```

Comment:
Are ID's required now?

Reply:
They will be required in the next version,
Currently it just raises an unpleasant deprecation warning each render